src/Forms/Forms.Component.js

```diff
@@ -92,21 +92,18 @@ export const FormsComponent = () => {
 
             <Example
                 description={`The state of the input field can reflect validity of the data entered, 
-                    whether the input data is editable or disabled.\n\n* **Default**: The field is 
+                    whether the input data is editable or disabled.\n\n* **Normal**: The field is 
                     editable but no validation has occurred. \n\n* **Valid**: The data format entered 
                     has been validated and it’s correct, such as an email address.\n\n* **Invalid**: The 
                     data entered is not valid and must be corrected.\n\n* **Warning**: The data entered 
                     is formatted correctly but there are other issues are problematic but will not stop 
-                    the user from moving forward.\n\n* **Disabled**: This indicates the field is not 
-                    editable. A common use case is that this field is dependent on a previous entry or 
-                    selection within the form.\n\n* **Read Only**: Used to display static information 
-                    in the context of a form.\n\nAlong with Invalid and Warning, error messages should 
+                    the user from moving forward.\n\nAlong with Invalid and Warning, error messages should 
                     be displayed below the field so the user can correct the error and move forward.`}
                 title='Input States'>
                 <div>
                     <FormSet>
                         <FormItem>
-                            <FormLabel htmlFor='OatmD552'>Default Input</FormLabel>
+                            <FormLabel htmlFor='OatmD552'>Normal Input</FormLabel>
```

Comment:
I think i see what you are trying to do here - "default" is simply an unexpressed type, How about "Input (no state)"?

Reply:
Yeah I really don't like calling it "normal", but we do have a "normal" state and fundamentals refers to defaults as "normal". We have this in a couple other places to, so I'd almost rather have a cleanup story to make it consistent throughout the components.